Ginger/GingerCore/NoSqlBase/GingerMongoDb.cs

```diff
@@ -251,11 +252,11 @@ private string GetQueryParamater(string inputSQL,string param)
         }
         public override void PerformDBAction()
         {
-            if (!Connect())
-            {
-                Act.Error = "Failed to connect to Mongo DB";
-                return;
-            }
+            //if (!Connect())
+            //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5368638)